modules/installation-special-config-butane.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-butane_{context}"]
+= Creating machine configs with Butane
+
+Machine configs are used to configure control plane and worker machines by telling machines how to create users and filesystems, set up the network, install systemd units, and more.
+
+Because modifying machine configs can be difficult, you can use Butane configs to create machine configs for you, thereby making node configuration much easier.
+
+== About Butane
+
+Butane is a command-line utility that {product-title} uses to provide convenient short-hand syntax for writing machine configs, as well as for performing additional validation of machine configs. The format of the Butane config file that Butane accepts is defined in the
+https://coreos.github.io/butane/config-openshift-v4_8/[OpenShift Butane config spec].
+
+[id="rhcos-installing-butane_{context}"]
+== Installing Butane
+
+You can install the Butane tool (`butane`) to create {product-title} machine configs from a command-line interface. You can install `butane` on Linux, Windows, or macOS by downloading the corresponding binary file.
+
+[TIP]
+====
+Butane releases are backwards-compatible with older releases and with the Fedora CoreOS Config Transpiler (FCCT).
+====
+
+.Procedure
+
+. Navigate to the Butane image download page at https://mirror.openshift.com/pub/openshift-v4/clients/butane/.
+. Get the `butane` binary:
+.. For the newest version of Butane, save the latest `butane` image to your current directory:
++
+[source,terminal]
+----
+$ curl https://mirror.openshift.com/pub/openshift-v4/clients/butane/latest/butane --output butane
+----
++
+.. Optional: For a specific type of architecture you are installing Butane on, such as aarch64 or ppc64le, indicate the appropriate URL. For example:
++
+[source,terminal]
+----
+$ curl https://mirror.openshift.com/pub/openshift-v4/clients/butane/latest/butane-aarch64 --output butane
+----
+
+. Move the `butane` binary file to a directory on your `PATH`.
++
+To check your `PATH`, open a terminal and execute the following command:
++
+[source,terminal]
+----
+$ echo $PATH
+----
++
+. Make the downloaded binary file executable:
```

Comment:
Let's move this before the previous step, since `chmod +x butane` assumes the binary is still in the current directory and we've just moved it out.

Reply:
Good call!